README.md

```diff
@@ -18,7 +18,10 @@ Project is created with:
 	
 ## Deploy
 To run this project, install and run it locally using maven:
-
 ```
 mvn clean install
-```
\ No newline at end of file
+```
+# run integration test
+```
+mvn clean verify -Pintegration_test -Djdbc.properties=test-jdbc-connection
```

Comment:
Because spring lets to set property source name with an environment variable like this
@PropertySource("classpath:${jdbc.properties:jdbc-connection}.properties")
but maven don't let to set an environment variable

Reply:
Why do you need environment variable for properties? Different profiles leads to different configurations and even different beans in app context.
Provide me any reasons why is it better?

Reply:
I have only one configuration for tests and production, to use the same configuration with different profiles I use an environment variable in PropertySource annotation

Reply:
As far as I remember you told that any test configuration requires additional DB initialization functionality. As a result, integration test configuration can't be the same as production ready.
Парируй.

Reply:
I've found DataSourceInitializer for spring java config so to create an in-memory h2 database instead of a production database I need to more beans and different property. For these two beans I set profile and to set different property I use an environment variable

Reply:
This bean required only for `integrationTest` profile. Am I right?

Reply:
yes

Reply:
So what the problem to mark it as `@Profile("integrationTest")`?

Reply:
I've removed redundant property -Djdbc.properties=test-jdbc-connection